packages/zoe/src/zoeService/types.ts

```diff
@@ -115,7 +123,7 @@ export type InstallBundle = (
  * Create an installation from a Bundle ID. Returns an installation.
  */
 export type InstallBundleID = (
-  bundleID: BundleID,
+  bundleID: string,
```

Comment:
this looks like an unfortunate regression... or ... was `BundleID` previously `any` here?

Reply:
> the type couldn't resolve.

That's the sort of thing I had in mind when asking whether it was previously `any`.

https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/src/types-external.js#L169 doesn't have any non-trivial JSDoc, but https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/bundles.md#bundleids has a couple paragraphs of documentation.

I'm pretty sure there's a `b0-` thing too, though the swingset docs seem to exclude it.

I think some sort of `// XXX reconnect to BundleID type docs` marker is worthwhile. Not critical.

Reply:
`b1-${string}`.

Reply:
My comment was written before I saw Dan's. I was merely pointing out the dash in the pattern, not responding to the possibility of `b0-`.  I think `b0-` is used in different places, but I'd check with @kriskowal  to be sure.